### ISSUE TITLE
woe_ply转换时，不能将缺失值转换成其对应woe

### ISSUE THREAD

**Issue by ChenSir99** [ShichenXie/scorecardpy#103]
woe_ply转换时，不能将缺失值转换成缺失值所在箱对应的woe，而是转换成了一个非缺失值箱对应的woe值，请问该怎么解决

**Comment by ShichenXie** [comment]
不会吧，能给个重现的例子么

**Comment by ShichenXie** [comment]
你先更新到github上面的版本试试呢

**Comment by ChenSir99** [comment]
[](url)
![微信图片_20230516181252](https://github.com/ShichenXie/scorecardpy/assets/92529355/17c9e789-d451-41d8-b6ff-5744cd982377)
缺失值这箱的woe是-0.39，另外一箱的woe是0.50。但是用woe_ply转化后，就全变成0.50了。
代码是
bins = sc.wobin(train,y='tag',method='chimerge',special_values=[-999,-1111])
train_woe = sc.woebin_ply(train,bins)
（上周刚从github上下载的包）

**Comment by ChenSir99** [comment]
[](url)
![微信图片_20230516181257](https://github.com/ShichenXie/scorecardpy/assets/92529355/19847039-1937-4952-b4ed-353bb1a2aa61)

**Comment by ShichenXie** [comment]
有数据和对应代码么，能让我在本地电脑复现的

**Comment by shuaihangs** [comment]
Yeah same issue, I have the same problem where WOE values are not applied to the special values. This occurs when there is a -inf to inf for one bin and all other bins are special values like -9999, -999999, -999999999 where these values satisfy the condition -info to inf, but infact they have their own "special value" bin. 

感觉像是负无穷到正无穷的优先级更高...
